src/main/java/Figure.java

```diff
@@ -0,0 +1,20 @@
+public abstract class Figure implements Draw {
```

Comment:
AreaCalculator ?

Reply:
What about this comment. Lets create AreaCalculator interface with getArea() method instead of define this method in Figure class